From 8b854b4064854332afa70decd4d66eb60aa984ac Mon Sep 17 00:00:00 2001
From: anonym <anonym@riseup.net>
Date: Thu, 15 Oct 2020 14:34:50 +0200
Subject: [PATCH] Disable protectHeaders AKA MemoryHole.

We're not sure it's ready for prime-time yet. Details:

* https://gitlab.tails.boum.org/tails/tails/-/issues/15201
* https://gitlab.tails.boum.org/tails/tails/-/issues/13649
---
 .../extensions/openpgp/content/ui/enigmailMsgComposeOverlay.js  | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/comm/mail/extensions/openpgp/content/ui/enigmailMsgComposeOverlay.js b/comm/mail/extensions/openpgp/content/ui/enigmailMsgComposeOverlay.js
index 39e6604a00e..5f95552f71b 100644
--- a/comm/mail/extensions/openpgp/content/ui/enigmailMsgComposeOverlay.js
+++ b/comm/mail/extensions/openpgp/content/ui/enigmailMsgComposeOverlay.js
@@ -138,7 +138,7 @@ Enigmail.msg = {
   composeBodyReady: false,
   modifiedAttach: null,
   lastFocusedWindow: null,
-  protectHeaders: true,
+  protectHeaders: false,
   draftSubjectEncrypted: false,
   attachOwnKeyObj: {
     attachedObj: null,
-- 
2.28.0

